terminus-ui/drawer/src/drawer-container.component.spec.ts

```diff
@@ -0,0 +1,218 @@
+import { PlatformModule } from '@angular/cdk/platform';
+import { ScrollingModule } from '@angular/cdk/scrolling';
+import { CommonModule } from '@angular/common';
+import { Type } from '@angular/core';
+import {
+  async,
+  ComponentFixture,
+  fakeAsync,
+  flush,
+  tick,
+} from '@angular/core/testing';
+import {
+  FormsModule,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { MatRippleModule } from '@angular/material/core';
+import { By } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { createComponent as createComponentInner } from '@terminus/ngx-tools/testing';
+import * as testComponents from '@terminus/ui/drawer/testing';
+import { TsDrawerComponent } from './drawer.component';
+import { TsDrawerModule } from './drawer.module';
+
+function createComponent<T>(component: Type<T>): ComponentFixture<T> {
+  const moduleImports = [
+    BrowserAnimationsModule,
+    CommonModule,
+    FormsModule,
+    ReactiveFormsModule,
+    MatRippleModule,
+    ScrollingModule,
+    PlatformModule,
+    TsDrawerModule,
+  ];
+
+  return createComponentInner(component, undefined, moduleImports);
+}
+
+describe(`TsDrawerContainerComponent`, () => {
+  let fixture;
+  let component;
+  let drawers;
+
+  function setUpTestComponent(testComponent) {
+    fixture = createComponent(testComponent);
+    fixture.detectChanges();
+    component = fixture.debugElement.queryAll(By.css('ts-drawer-container'))[0].componentInstance;
```

Comment:
I don't think we should delay this PR any more, but for future reference, I think we should abstract the queries you need into that test-helpers file so that consumers and our other specs can use them too.